src/main/java/raythx98/duke/parser/Parser.java

```diff
@@ -0,0 +1,104 @@
+package raythx98.duke.parser;
+
+import raythx98.duke.command.AddCommand;
+import raythx98.duke.command.AddDeadlineCommand;
+import raythx98.duke.command.AddEventCommand;
+import raythx98.duke.command.AddTodoCommand;
+import raythx98.duke.command.Command;
+import raythx98.duke.command.DeleteCommand;
+import raythx98.duke.command.DoneCommand;
+import raythx98.duke.command.ExitCommand;
+import raythx98.duke.command.FindCommand;
+import raythx98.duke.command.InvalidCommand;
+import raythx98.duke.command.ListCommand;
+import raythx98.duke.exception.DukeException;
+import raythx98.duke.exception.EmptyDescriptionException;
+import raythx98.duke.exception.InvalidIndexException;
+import raythx98.duke.exception.UnknownCommandException;
+import raythx98.duke.task.Task;
+import raythx98.duke.task.TaskList;
+
+/**
+ * Deals with making sense of user inputs.
+ */
+public class Parser {
+
+    /**
+     * Parses the user data by making processing input.
+     *
+     * @param tasks    List of tasks at the moment.
+     * @param nextLine String to be parsed.
+     * @return
```

Comment:
Is there a missing statement within the Javadoc comment to describe the what the method returns?

Reply:
Agreed, will resolve